### PR TITLE
Enable CBO on Azure and GCP platforms

### PR DESCRIPTION
**Pull request by elfosardo**
Enabling CBO on Azure and GCP platforms allows a user to deploy and manage bare metal clusters using a true hybrid cloud solution.

This enhancement uses and expands the work done for https://github.com/openshift/enhancements/blob/master/enhancements/baremetal/enable-baremetal-on-other-platforms.md https://github.com/openshift/enhancements/blob/master/enhancements/baremetal/enable-baremetal-on-aws.md

### CODE REVIEW

File: enhancements/baremetal/enable-baremetal-on-azure-gcp.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+QE will validate that the Azure and GCP platforms are supported to reduce the load on CI.
+
+### Graduation Criteria
+
+#### Dev Preview -> Tech Preview
+
+#### Tech Preview -> GA
+
+The feature was silently enabled in Openshift 4.14 and it will go to GA in Openshift 4.14.
```
Comment (dtantsur) [inline enhancements/baremetal/enable-baremetal-on-azure-gcp.md:137]: Copy-paste from AWS, does not make sense here.

Comment (elfosardo) [reply enhancements/baremetal/enable-baremetal-on-azure-gcp.md:137]: whoops forgot to change that

Comment (elfosardo) [reply enhancements/baremetal/enable-baremetal-on-azure-gcp.md]: I fixed the versions, should be ok now

**Pull request merged**